lib/msal-common/src/request/broker/BrokeredSilentFlowRequest.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { SilentFlowRequest } from "../SilentFlowRequest";
+
+/**
+ * SilentFlow parameters passed by the user to retrieve credentials silently
+ * - scopes                 - Array of scopes the application is requesting access to.
+ * - claims                 - A stringified claims request which will be added to all /authorize and /token calls. When included on a silent request, cache lookup will be skipped and token will be refreshed.
+ * - authority              - Url of the authority which the application acquires tokens from.
+ * - correlationId          - Unique GUID set per request to trace a request end-to-end for telemetry purposes.
+ * - account                - Account entity to lookup the credentials.
+ * - forceRefresh           - Forces silent requests to make network calls if true.
+ * - resourceRequestMethod      - HTTP Request type used to request data from the resource (i.e. "GET", "POST", etc.).  Used for proof-of-possession flows.
+ * - resourceRequestUri         - URI that token will be used for. Used for proof-of-possession flows.
+ */
+export type BrokeredSilentFlowRequest = SilentFlowRequest & {
```

Comment:
Do these need to live in common? Or can they live in browser (until we know broker is needed for something like Node)?

Reply:
They need to be in common because common needs to know about the embeddedAppClientId and brokerRedirectUri. There isn't really a way around that, and I'm not a fan of making them one-off function parameters, especially if things start getting added to the broker flow.